### ISSUE TITLE
Namespace completion won't trigger between dots

### ISSUE THREAD

**Issue by rchande** [dotnet/roslyn#35236]
```C#
namespace N.O.P
{
}

namespace N.$$.P
{
}
```

Intellisense doesn't trigger on ctrl-space at $$ unless a character is typed, eg:
![image](https://user-images.githubusercontent.com/3751401/56677205-dc504b80-6674-11e9-9c9a-0f7944f5e1d5.png)
